example.py

```diff
@@ -30,3 +30,7 @@
 # Get all available voices and print the first one
 all_voices = aflr.Voice().list()
 print(all_voices["voices"][0])
+
+# Get all available voices and print the first one
+sync_tts = aflr.SyncTTS().create(text="Hello synchronous text to speech", voice="voice id")
```

Comment:
Can we have a flag or something - that you need an id to call it. We could give it to say Uneeq but not others. 

Do we have feature flags?

Reply:
we can enable voice ids only for uneeq if we want to